integrations/instructor_embedders/pyproject.toml

```diff
@@ -152,13 +154,13 @@ unfixable = [
   "F401",
 ]
 
-[tool.ruff.isort]
+[tool.ruff.lint.isort]
 known-first-party = ["instructor_embedders"]
```

Comment:
Shouldn't we have `haystack_integrations` here too?

Reply:
:+1: